rep-0003.rst

```diff
@@ -233,6 +233,66 @@ Buildsystem support:
 
   - build from source
 
+Lunar Loggerhead (May 2017 - May 2019)
+--------------------------------------
+Required Support for:
+
+- Ubuntu Xenial (16.04)
+- Ubuntu Yakkety (16.10)
+- Ubuntu Zesty (17.04)
+
+Recommended Support for:
+
+- Debian Stretch
+- Fedora 25
+
+Architecture Supported:
+
+- amd64
+- arm32
+- arm64
+
+Minimum Requirements:
+
+- C++11
+
+  - GCC 4.9 on Linux, as it's the version that Debian Jessie ships with
```

Comment:
the goal is to avoid people from branching out before the next LTS and to reduce the amount of effort for users wanting to build on Jessie even if it's not listed as a supported platform. Maybe I should just say "same as Kinetic" for all of them